tests/utils.py

```diff
@@ -0,0 +1,23 @@
+from contextlib import contextmanager
+from typing import Any, Dict, Union
+
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.namespace import Namespace
+
+TIMEOUT_2MIN = 2 * 60
+
+
+@contextmanager
+def create_ns(
```

Comment:
Can you take a look at https://github.com/opendatahub-io/opendatahub-tests/pull/22/files#diff-f828bbdfd43e1482ca6af2ca3b68408b15bbbbbd519fc71fdc94e02dc559c5eeR7 and see if we can merge the two? 
e.g. in my case I type the labels to be Optional instead of doing the union between a Dict and a None, I wait for the ns to be active etc.